addons/survey/static/src/js/survey_quick_access.js

```diff
@@ -36,6 +40,20 @@ publicWidget.registry.SurveyQuickAccessWidget = publicWidget.Widget.extend({
     // Handlers
     // -------------------------------------------------------------------------
 
+    _launch_session: function () {
+        this._rpc({
```

Comment:
We try to use ES6 syntax for those.

So you will have:

```js
const sessionResult = await this.rpc({...});
window.location = sessionResult.url;
```

Not sure it's worth to test that you have "url" in the result as this is the only possible response (outside of a complete failure but then reloading the page is not going to help).

(You might have to make your function async for that)